packages/react/src/components/TextField/TextField.test.tsx

```diff
@@ -527,78 +483,86 @@ describe('TextField with error message', () => {
     );
     jest.runAllTimers();
     expect(validator).toHaveBeenCalledTimes(0);
-    assertErrorMessage(wrapper.getDOMNode(), false);
+    assertErrorMessage(container, false);
 
-    wrapper.setProps({ value: 'failValidationValue' });
+    rerender(
+      <TextField
+        validateOnFocusOut
```

Comment:
Up to you but at least in my opinion, with this many props it might be good to save the ones that don't change in an object and spread it each time.

Reply:
Sounds good to me, will make the change!